docs/framework-integration/angular.md

```diff
@@ -309,15 +305,58 @@ export class AppModule {}
 
 You can now directly leverage your components in their template and take advantage of Angular template binding syntax.
 
-```html
-<!-- app.component.html -->
-
+```html title="app.component.html"
 <my-component first="Your" last="Name"></my-component>
 ```
 
 From your Angular workspace (`/packages/angular-workspace`), run `npm start` and navigate to `localhost:4200`. You should see the
 component rendered correctly.
 
+### Angular with Standalone Components
+
+In Angular CLI v17, the default behavior is to generate a new project with standalone components.
+
+From your Angular workspace (`/packages/angular-workspace`), run the following command to generate an Angular application:
+
+```bash
+npx -p @angular/cli ng generate app my-app
+```
+
+#### Consuming the Angular Wrapper Components
+
+This section covers how developers consuming your Angular component wrappers will use your package and component wrappers.
+
+In order to use the generated component wrappers in the Angular app, you'll first need to build your Angular component library. From the root
+of your Angular workspace (`/packages/angular-workspace`), run the following command:
+
+```bash
+npx -p @angular/cli ng build component-library
+```
+
+Now you can import and reference your components in your consuming application in the same way you would with any other Angular components:
+
+```ts title="app.component.ts"
+import { Component } from '@angular/core';
+import { MyComponent } from 'component-library';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+  standalone: true,
+  imports: [MyComponent],
+})
+export class AppComponent {}
```

Comment:
Using this snippet, I ran into the following error message when trying to run hte local dev server for the app:
```
[ERROR] TS-992011: The component 'MyComponent' appears in 'imports', but is not standalone and cannot be imported directly. It must be imported via an NgModule. [plugin angular-compiler]

    projects/my-app/src/app/app.component.ts:9:12:
      9 │   imports: [MyComponent],
        ╵             ~~~~~~~~~~~
```

I ran through the instructions here from start to end to verify that I wasn't starting with a pre-existing reproduction locally that was in a bad state. 

Do we need to call out to folks that when they need to use 'standalone' for the [outputType](https://stencil-docs-git-sp-fw-5678-ionic1.vercel.app/docs/next/angular#outputtype) when they're configuring the Stencil OT here? I _think_ that'd lead to quite a few more changes, but I don't fully understand the error message above and wanted to double check before we went down that road

Reply:
Ahh yes, we need to additionally reference each usage for the different `outputType` configurations. I'll think on how we might split this up. I think separating consumer usage by type makes sense, but maybe another subsection within it for the different output types.